### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jsimonclark**
add general description of the redirection rules to the readme

### CODE REVIEW

**Comment by jsimonclark** [comment]
@jesper-friis , a few questions

- what do you mean with rules 1 and 3? There are no {DOMAIN} tags there. 
- what is the difference between rules 6 and 8?

I'm sure there is some reason behind those - but can we make it more clear in the explanation in the readme?

File: EMMO/README.md [hunk 1]
```diff
@@ -1,6 +1,49 @@
 # EMMO
 This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Elementary Multiperspective Material Ontology (EMMO)](https://github.com/emmo-repo/EMMO/) and its domain ontologies.
 
+## Redirection Rules
+This section contains a general summary of the logic behind the redirection rules.  
+
+1. `https://w3id.org/emmo`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch
```
Comment (jesper-friis) [inline EMMO/README.md:8]: There is no DOMAIN here, so this should redirect to emmo. Probably it make sense to treat it as a shorthand for `https://w3id.org/emmo/emmo`. 

Should we use [IndustryPortal](http://industryportal.enit.fr/ontologies/EMMO?p=classes&conceptid=http%3A%2F%2Femmo.info%2Femmo%23EMMO_802d3e92_8770_4f98_a289_ccaaab7fdddf) or GitHub Pages if the user is accessing from a browser?

If the user is not accessing from a browser, maybe it make more sense to redirect to a squashed ontology (all modules combined in a single ttl file for much faster access) on GitHub Pages?

File: EMMO/README.md [hunk 2]
```diff
@@ -1,6 +1,49 @@
...
 This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Elementary Multiperspective Material Ontology (EMMO)](https://github.com/emmo-repo/EMMO/) and its domain ontologies.
 
+## Redirection Rules
+This section contains a general summary of the logic behind the redirection rules.  
+
+1. `https://w3id.org/emmo`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+2. `https://w3id.org/emmo#{NAME}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch
```
Comment (jesper-friis) [inline EMMO/README.md:11]: Same as for rule 1.

File: EMMO/README.md [hunk 3]
```diff
@@ -1,6 +1,49 @@
...
+This section contains a general summary of the logic behind the redirection rules.  
+
+1. `https://w3id.org/emmo`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+2. `https://w3id.org/emmo#{NAME}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+3. `https://w3id.org/emmo/`
+   - `{DOMAIN}.ttl` file in the root of the master branch
```
Comment (jesper-friis) [inline EMMO/README.md:14]: emmo.ttl file in the root of the master branch

File: EMMO/README.md [hunk 4]
```diff
@@ -1,6 +1,49 @@
...
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+2. `https://w3id.org/emmo#{NAME}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+3. `https://w3id.org/emmo/`
+   - `{DOMAIN}.ttl` file in the root of the master branch  
+
+4. `https://w3id.org/emmo/{VERSION}`
+   - Given version of EMMO
```
Comment (jesper-friis) [inline EMMO/README.md:17]: Would it make sense to treat the same way as rule 1, except that it redirects to the given version?

File: EMMO/README.md [hunk 5]
```diff
@@ -1,6 +1,49 @@
...
+   - `{DOMAIN}.ttl` file in the root of the master branch  
+
+4. `https://w3id.org/emmo/{VERSION}`
+   - Given version of EMMO  
+
+5. `https://w3id.org/emmo/{VERSION}/{PATH}/{MODULE}`
+   - Given version and module of EMMO  
+
+6. `https://w3id.org/emmo/{DOMAIN}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch
```
Comment (jesper-friis) [inline EMMO/README.md:23]: If user is not accessing from a browser, should the redirection go to a squashed .ttl file in github pages instead?

File: EMMO/README.md [hunk 6]
```diff
@@ -1,6 +1,49 @@
...
+   - Given version of EMMO  
+
+5. `https://w3id.org/emmo/{VERSION}/{PATH}/{MODULE}`
+   - Given version and module of EMMO  
+
+6. `https://w3id.org/emmo/{DOMAIN}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+7. `https://w3id.org/emmo/{DOMAIN}#{NAME}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch
```
Comment (jesper-friis) [inline EMMO/README.md:26]: Same as rule 6

**Comment by jesper-friis** [comment]
Thanks Simon. I will merge this PR and make a new one where I try to address the above comments. I hope it will make the iterations easier.

**Pull request merged**